app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/securities/CheckClaimDetailsController.scala

```diff
@@ -72,11 +71,8 @@ class CheckClaimDetailsController @Inject() (
           journey,
           if (userHasSeenCYAPage(journey))
             Redirect(routes.CheckYourAnswersController.show())
-          else if (journey.requiresDocumentTypeSelection) {
+          else
             Redirect(routes.ChooseFileTypeController.show())
```

Comment:
should it be `CheckBankDetailsController.show()`?

Reply:
Yes of course. Got it mixed up in some last minute changes. Will fix in a sec.